rfc/rfc014-authorization-credential.md

```diff
@@ -0,0 +1,219 @@
+
+### Copyright Notice
+![](../.gitbook/assets/license.png)
+
+This document is released under the [Attribution-ShareAlike 4.0 International \(CC BY-SA 4.0\) license](https://creativecommons.org/licenses/by-sa/4.0/).
+
+## 1. Introduction
+
+Personal data (this includes medical data) is protected under the GDPR and various local legislation. 
+It's common in healthcare to receive care from multiple care organizations. 
+These organizations may not share data amongst them without a valid legal reason. 
+The GDPR sums up several legal reasons to share data. Most common are local legislation and explicit consent. 
+When data is shared between two organizations, this only matters to those organizations (and the patient). 
+Others are not allowed to see this relation. 
+Simply knowing where someone receives care may give away the illness of the patient.
+The type of data that is shared should also be limited to what is needed by the actor.
+
+There are various reasons why data can be shared. 
+The process of determining a valid reason happens at different places and at different times:
+
+- A patient at the GP's office wants the GP to view data from the hospital.
+- A patient consents to a specific treatment or test and this treatment/test requires specific data.
+- An elderly receiving home care wants his/her GP to be able to review data from the home care organization.
+- A patient is discharged from the hospital and needs to undergo physical therapy. 
+- Etc.
+
+This RFC builds upon [RFC011](rfc011-verifiable-credential.md).
+
+## 2. Terminology
+
+* **DID**: [Decentralized Identifiers](https://www.w3.org/TR/did-core/).
+* **Policy**: A security policy defined by a Bolt. It describes the access to and operations on resources that are allowed. 
+* **VC**: [Verifiable Credential Data Model](https://www.w3.org/TR/vc-data-model/).
+
+Other terminology comes from the [Nuts Start Architecture](rfc001-nuts-start-architecture.md#nuts-start-architecture).
+
+## 3. CredentialSubject
+
+The `credentialSubject` field contains the following:
+
+```json
+{
+  "id": "did:nuts:SjkuVHVqZndMVVJwcnUzbjhuZklhODB1M1M0LW9LcWY0WUs5S2",
+  "legalBase": {
+    "consentType": "explicit",
+    "evidence": {
+      "path": "pdf/f2aeec97-fc0d-42bf-8ca7-0548192d4231",
+      "type": "application/pdf"
+    }
+  },
+  "restrictions": [
+    {
+      "resource": "/DocumentReference/f2aeec97-fc0d-42bf-8ca7-0548192d4231",
+      "operations": ["read"]
+    }
+  ],
+  "service": "test-service",
+  "subject": "urn:oid:2.16.840.1.113883.2.4.6.3:123456780"
+}
+```
+
+### 3.1 Required fields
+
+The `id`, `legalBase` and `service` fields MUST be filled. 
+Within the `legalBase` object, `consentType` MUST either be `implicit` or `explicit`. 
+When `explicit`, the `evidence` and `subject` fields MUST be filled.
+When `implicit`, values MUST be added to the `restrictions` array.
+
+### 3.2 Scoping
+
+The `issuer` field MUST contain the DID of the custodian.
```

Comment:
Issuer is not part of the credentialSubject (name of this chapter).

Reply:
I split chapter 3 into 3.1 and 3.2. Where 3.1 is about generic requirements on the credential and 3.2 is about the credentialSubject.